source/guide_phpservermonitor.rst

```diff
@@ -0,0 +1,114 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/phpservermonitor.png
+      :align: center
+
+##################
+PHP Server Monitor
+##################
+
+.. tag_list::
+
+PHP Server Monitor_ is an open source tool to monitor your servers and websites.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We’re using :manual:`PHP <lang-php>` in the stable version 7.2:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.2'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+To install PHP Server Monitor we download the current version from the github release. ``cd`` to your :manual:`DocumentRoot <web-documentroot>` so the zip file will be under your ``html``.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/phpservermon/phpservermon/releases/download/v3.4.5/phpservermon-3.4.5.zip
```

Comment:
you can use `https://github.com/phpservermon/phpservermon/archive/v3.4.5.tar.gz` here. the URL is shorter and it enables the next step.

Reply:
Oh, I see. The tar doesn't contain the deps, the zip does :roll_eyes: